data_processors/pipeline/lambdas/dragen_wgs_qc.py

```diff
@@ -103,13 +104,28 @@ def handler(event, context) -> dict:
 
     sample_name = fastq_list_rows[0]['rgsm']
 
-    # Set workflow helper
-    wfl_helper = SecondaryAnalysisHelper(WorkflowType.DRAGEN_WGS_QC)
+    # Get metadata by library id
+    library_lab_metadata = metadata_srv.get_metadata_by_library_id(library_id)
+
+    # We set the RNA flag and set the workflow type based on the library lab metadata
+    if library_lab_metadata.type == LabMetadataType.WTS:
```

Comment:
As discussed. The `metadata_srv.get_metadata_by_library_id(library_id)` could return `None` if not found in database.

Hence. We should better handle `if library_lab_metadata is None` before accessing its attribute `library_lab_metadata.type` value checking.

Reply:
Resolved in 7583771